td-layout/src/runtime.rs

```diff
@@ -54,4 +54,4 @@ pub const TD_HOB_SIZE: u64 = 0x20000;
 pub const TD_PAYLOAD_PARAM_BASE: u64 = 0x1100000;
 pub const TD_PAYLOAD_PARAM_SIZE: u64 = 0x100000;
 pub const TD_PAYLOAD_BASE: u64 = 0x1200000;
-pub const TD_PAYLOAD_SIZE: usize = 0x8000000;
+pub const TD_PAYLOAD_SIZE: usize = 0x2000000;
```

Comment:
why change it back?

Reply:
Didn't notice this, need to update config file for this because the previous commit only modified the `runtime.rs`, it will change back if we rerun the layout-config tool